.gitlab-ci.yml

```diff
@@ -63,3 +63,32 @@ build_wheel:
     # create the wheel and upload to pypi
     - sh scripts/create_assembl_wheel.sh
     - twine upload -u bluenove --skip-existing dist/*
+
+deploy_to_dev:
+  stage: deploy_to_dev
+  only:
+    - tmp_master_for_ci_cd
+    - master
```

Comment:
@MilanBarande you should remove `master` from now

Reply:
plus logique :)